hanko-js/README.md

```diff
@@ -0,0 +1,81 @@
+# &lt;hanko-auth&gt; element
+
+The `<hanko-auth>` element offers a complete user interface that will bring a modern
+login and registration experience to your users. It integrates the Hanko API,
+a backend that provides the underlying functionalities.
+
+## Features
+
+* Registration and login flows with and without passwords
+* Platform Authenticators (e.g. Apple's Passkeys, Windows Hello, etc.)
+* Passcodes, a convenient way to recover passwords and verify email addresses
+* Language support for English and German
+
+## Upcoming Features
+
+* Customizable UI styles
+* Support for Security Keys
+* Exponential backoff mechanisms
+* Testing and code documentation
+
+## Installation
+
+_WiP_
+
+## Usage
+
+### Script
+
+Import as a modules:
+
+_WiP_
+
+With a script tag via CDN:
+
+_WiP_
+
+### Markup
+
+```html
+<hanko-auth api="https://hanko.yourdomain.com" lang="en" />
+```
+
+Please take a look at the [Hanko API](../backend/README.md) to see how to spin up the backend.
+
+Note, that we're working on a SaaS solution, so that you don't need to run the
+Hanko API by yourself and everything you need is to add the `<hanko-auth>` element
+to your page.
```

Comment:
Note that we're working on Hanko Cloud, so that you don't need to run the
Hanko API by yourself and all you need is to do is adding the `<hanko-auth>` element
to your page.

Reply:
Thanks, updated!